public/js/action-list.js

```diff
@@ -1,6 +1,6 @@
 /* Icinga DB Web | (c) 2020 Icinga GmbH | GPLv2 */
 
-;(function () {
```

Comment:
Please keep the semicolon. JS files are concatenated into a single file. This helps isolating files.